frontend/public/components/RBAC/role.jsx

```diff
@@ -249,12 +276,27 @@ export const RolesDetailsPage = (props) => {
         { href: 'bindings', name: t('role~RoleBindings'), component: BindingsForRolePage },
       ]}
       menuActions={menuActions}
+      breadcrumbsFor={() => getBreadcrumbs('Role', props.kindObj, props.match)}
```

Comment:
Can't we use the default here? I don't think we need to override `breadcrumbsFor`.

Reply:
umm...we seem to be going in [circles](https://github.com/openshift/console/pull/8010#discussion_r570569030).

Reply:
I guess I'm confused. Is this component only used for Roles, or is it used for both Roles and ClusterRoles? I don't see where it's registered as the details page for ClusterRoles.

Reply:
It is used for Both [here](https://github.com/openshift/console/blob/cb0e42882c6ea3bb187f70bd96fd1c600c2c520c/frontend/public/components/RBAC/role.jsx#L284) and [here](https://github.com/openshift/console/blob/cb0e42882c6ea3bb187f70bd96fd1c600c2c520c/frontend/public/components/resource-pages.ts#L228).

Reply:
Thanks, that's what I missed. I was looking for it in resource-pages.ts.